pki.spec

```diff
@@ -858,6 +858,7 @@ fi
     -DWITH_SERVER:BOOL=OFF \
 %endif
     -DWITH_JAVADOC:BOOL=%{?with_javadoc:ON}%{!?with_javadoc:OFF} \
+    -DWITH_JAXB:BOOL=%{?is_java_11:ON}%{!?is_java_11:OFF} \
```

Comment:
Ah yuck. The [changes from v10.9](https://github.com/dogtagpki/pki/blob/v10.9/pki.spec#L59) didn't get pulled in correctly. Could you unify the spec file to `v10.9` level and then update this?

Reply:
OK, let me update.